docs/events/Idaho_Pilot_2022.md

```diff
@@ -43,4 +43,4 @@ MITRE wrote an independent verifier for this election to ensure that ElectionGua
 
 <!-- Links -->
 [Idaho-Banner]: /images/ElectionGuard-Idaho-2022.svg "Idaho Banner"
-[hart-scanner]: /images/VotingMachine.jpg
+[hart-scanner]: /images/votingmachine.jpeg
```

Comment:
@jess24454 Did you mean to change this?  I may be missing something, but it looks like the file name is still `votingmachine.jpg`, so this change would break the link to the image.

Reply:
@lprichar The original filename had a capitalized extension JPG.  when Jessica went to fix it, Windows of course did not tell that there was a difference between JPG and jpg.  And the makedocs system lowercased it all anyways.  So to fix it I had her rename the file so that we can get Windows and makedocs together with a new name.